packages/terra-tabs/src/Tabs.jsx

```diff
@@ -189,6 +189,8 @@ class Tabs extends React.Component {
       </CollapsedTabs>
     );
 
+    debugger;
```

Comment:
Looks like there is a debugger statement that needs removed.

Reply:
I was debugging Tabs and didn't realize the change got commited. Removed. https://github.com/cerner/terra-core/pull/1155/commits/81a22fc16181f57f506752dc4df5abb6c6eeea55